indico/modules/events/persons/controllers.py

```diff
@@ -347,3 +359,54 @@ def _process(self):
             tpl = get_template_module('events/persons/management/_person_list_row.html')
             return jsonify_data(html=tpl.render_person_row(person_data, bool(self.event.registration_forms)))
         return jsonify_form(form)
+
+
+class RHEventPersonSearch(RHPersonsBase):
+    def _search_event_persons(self, exact=False, **criteria):
+        criteria = {key: value.strip() for key, value in criteria.items() if value.strip()}
```

Comment:
I think you can use some Python 3.9 magic here ;)

```suggestion
        criteria = {key: v for key, value in criteria.items() if (v := value.strip())}
```

Nothing wrong with your solution though, stripping is cheap!